src/config.rs

```diff
@@ -98,6 +98,10 @@ pub struct Config {
 
     /// Max size for committed entries in a `Ready`.
     pub max_committed_size_per_ready: u64,
+
+    /// Maximum raft log number that can be applied after commit but before persist.
+    /// The default value is 0, which means apply after both commit and persist.
+    pub apply_unpersisted_log_limit: u64,
```

Comment:
Will it support dynamic config update?

Reply:
Yes, we provide a public function `Raft::set_apply_unpersisted_log_limit` to change this value in realtime.